rust/foxglove/src/lib.rs

```diff
@@ -1,3 +1,159 @@
+//! The official [Foxglove] SDK.
+//!
+//! This crate provides support for integrating with the Foxglove platform. It can be used to log
+//! events to local [MCAP] files or a local visualization server that communicates with the
+//! Foxglove app.
+//!
+//! [Foxglove]: https://docs.foxglove.dev/
+//! [MCAP]: https://mcap.dev/
+//!
+//! # Getting started
+//!
+//! To record messages, you need at least one sink, and at least one channel. In this example, we
+//! create an MCAP file sink, and a channel for [`Log`](`crate::schemas::Log`) messages on a topic
+//! called `"/topic"`. Then we write one log message and close the file.
```

Comment:
Uup, my bad on the naming there.

```suggestion
//! called `"/log"`. Then we write one log message and close the file.
```